worlds/kh2/Client.py

```diff
@@ -589,8 +615,9 @@ async def IsInShop(self, sellable):
         # if journal=-1 and shop = 5 then in shop
         # if journal !=-1 and shop = 10 then journal
 
-        journal = self.kh2_read_short(0x741230)
-        shop = self.kh2_read_short(0x741320)
+        #todo:get these addresses updated
```

Comment:
Is this done now?

Reply:
yeah its done and people have been running on this version of the client for 4 days any not a single person has had issues with the updated memory addresses or steam support